src/app/app.component.scss

```diff
@@ -247,6 +247,7 @@ ul.navigation-primary {
 
 .help-menu > .menu-trigger {
   background: transparent;
+  font-size: 14px;
```

Comment:
This should be the same change as in the .info-card > .menu-trigger` of the dashboard so the **?** and **i** use the same formatting.

Reply:
Done